WordPress/src/main/res/layout/domain_suggestions_fragment.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              xmlns:app="http://schemas.android.com/apk/res-auto"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:background="@color/white"
+              android:focusable="true"
+              android:orientation="vertical">
+
+    <LinearLayout
+        android:id="@+id/searchFieldContainer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_extra_small"
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        android:padding="@dimen/margin_extra_large">
+
+        <FrameLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/margin_large">
+
+            <android.support.v7.widget.AppCompatImageView
```

Comment:
```suggestion
            <ImageView
```

The `android.support.v7.widget.AppCompatImageView` can be changed to `ImageView`.  I don't see it used anywhere else in the codebase.  According to the [`AppCompatImageView` documentation](https://developer.android.com/reference/android/support/v7/widget/AppCompatImageView):
> [`AppCompatImageView`] will automatically be used when you use `ImageView` in your layouts and the top-level activity / dialog is provided by appcompat. You should only need to manually use this class when writing custom views.